### PR TITLE
add docs (work in progress)

### PR DESCRIPTION
**Pull request by jwld**
relates #73

### CODE REVIEW

File: docs.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# Open Tourism Platform API
+An API providing data for the four apps on the Nazareth Open Tourism Platform, built by graduates of [Founders and Coders](https://foundersandcoders.com/about/).
+
+## Contents
+**Events**
+- [Get all events](#Get all events)
```
Comment (m4v15) [inline docs.md]: these links don't work

Comment (m4v15) [reply docs.md]: need no capitals and hyphen instead of spaces in the brackets I think

Comment (jwld) [reply docs.md]: thanks, fixed 👍

File: docs.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+### Create event
+`POST /events`
+
+**Input**
+
+Name | Type | Description
+---|---|---
+name | string | **Required**. The name of the event.
+category | array of strings | **Required**. The event category / categories.
+accessibilityOptions | array of strings | Event accessibility options.
```
Comment (des-des) [inline docs.md:79]: I think these may be restricted.

See here: https://github.com/foundersandcoders/open-tourism-platform/blob/master/src/models/constants.json

These will prob change, for now you could just drop in a link

File: docs.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  "category": [
+    "music"
+  ]
+}
+```
+
+### Create event
+`POST /events`
+
+**Input**
```
Comment (des-des) [inline docs.md:73]: Github called them parameters.

https://developer.github.com/v3/repos/#parameters

Not sure what is better

Comment (jwld) [reply docs.md:73]: Create and update both refer to them as 'input' however: 

https://developer.github.com/v3/repos/#create

Comment (des-des) [reply docs.md:73]: :+1:

**Review by des-des — APPROVED** [review]

**Comment by m4v15** [comment]
@des-des, it was work in progress, it wasn't right

**Comment by m4v15** [comment]
The objects that are returned now have ```en``` and ```ar``` keys for exampe

**Comment by des-des** [comment]
derp

**Comment by des-des** [comment]
sorry

**Pull request merged**